docs/installation.md

```diff
@@ -11,21 +11,26 @@ To install `pixi` you can run the following command in your terminal:
     wget -qO- https://pixi.sh/install.sh | sh
     ```
 
-    The above invocation will automatically download the latest version of `pixi`, extract it, and move the `pixi` binary to `~/.pixi/bin`.
-    The script will also extend the `PATH` environment variable in the startup script of your shell to include `~/.pixi/bin`.
-    This allows you to invoke `pixi` from anywhere.
+    ??? note "What does this do?"
+        The above invocation will automatically download the latest version of `pixi`, extract it, and move the `pixi` binary to `~/.pixi/bin`.
+        The script will also extend the `PATH` environment variable in the startup script of your shell to include `~/.pixi/bin`.
+        This allows you to invoke `pixi` from anywhere.
 
 === "Windows"
     ```powershell
     powershell -ExecutionPolicy ByPass -c "irm -useb https://pixi.sh/install.ps1 | iex"
     ```
 
-    The above invocation will automatically download the latest version of `pixi`, extract it, and move the `pixi` binary to `%UserProfile%\.pixi\bin`.
-    The command will also add `%UserProfile%\.pixi\bin` to your `PATH` environment variable, allowing you to invoke `pixi` from anywhere.
+    ??? note "What does this do?"
+        The above invocation will automatically download the latest version of `pixi`, extract it, and move the `pixi` binary to `%UserProfile%\.pixi\bin`.
+        The command will also add `%UserProfile%\.pixi\bin` to your `PATH` environment variable, allowing you to invoke `pixi` from anywhere.
 
-!!! tip
+Now restart your terminal or shell to make the installation effective.
+
+??? question "Don't trust our link? Check the script!"
+    You can check the installation `sh` script: [download](https://pixi.sh/install.sh) and the `ps1`: [download](https://pixi.sh/install.ps1).
```

Comment:
as mentioned above, i would prefer not downloading those things but instead, just viewing them from my browser